tool/tsh/proxy.go

```diff
@@ -258,12 +267,6 @@ func dialSSHProxy(ctx context.Context, tc *libclient.TeleportClient, sp sshProxy
 	}
 
 	// Otherwise, we need to upgrade the TCP connection to a TLS connection.
```

Comment:
A bit confused here -- don't we already have a TLS connection in the case where `HTTPS_PROXY=https://something`?

Reply:
In that case there would be 2 TLS connections; one to the HTTP proxy server, and one inside that connection to the target server.